lib/pages/contacts/presentation/model/presentation_contact.dart

```diff
@@ -11,13 +11,16 @@ class PresentationContact extends Equatable {
 
   final ContactStatus? status;
 
+  final bool isOwner;
```

Comment:
Should be a get function with active client Id?

Reply:
didn't get your point

Reply:
this function maybe an extension, not all the case we need it :)